js/ui/welcomeDialog.js

```diff
@@ -32,7 +32,8 @@ class WelcomeDialog extends ModalDialog.ModalDialog {
     }
 
     _buildLayout() {
-        const title = _('Welcome to Endless OS 5');
+        const [majorVersion] = GLib.get_os_info('PRETTY_NAME').split('.');
```

Comment:
This does make an assumption about the format of PRETTY_NAME but I think it's fine.